boot/zephyr/main.c

```diff
@@ -40,6 +40,14 @@ const struct boot_uart_funcs boot_funcs = {
 };
 #endif
 
+#ifdef CONFIG_BOOT_WAIT_FOR_DFU
+#include <usb/class/usb_dfu.h>
+
+#define INTERMITTENT_CHECK_DELAY	50
```

Comment:
What is that - pleas putt comment in the code.

Reply:
Done.